Bottom: 62ab781e69bf98ac3a31bd51594adfb5e60693c2
Top:    21f3d60083bdf38fa71ed526586caa0855fc70d3
Author: Amit Kucheria <amit.kucheria@linaro.org>
Date:   2016-09-13 11:37:51 +0530

From f59a1c1351483847e3b6d2d340c663a924e78c24 Mon Sep 17 00:00:00 2001
Subject: [PATCH 06/21] pinmux: 96b_carbon: Setup SPI pins on the board

Change-Id: I739887d34f5eadb5475204b2b00130159de14347
Signed-off-by: Amit Kucheria <amit.kucheria@linaro.org>
Signed-off-by: Ricardo Salveti <ricardo.salveti@linaro.org>


---

diff --git a/drivers/pinmux/stm32/pinmux_board_carbon.c b/drivers/pinmux/stm32/pinmux_board_carbon.c
index 29a087221..eaa97ba7a 100644
--- a/drivers/pinmux/stm32/pinmux_board_carbon.c
+++ b/drivers/pinmux/stm32/pinmux_board_carbon.c
@@ -23,6 +23,18 @@ static const struct pin_config pinconf[] = {
 	{STM32_PIN_PA2, STM32F4_PINMUX_FUNC_PA2_USART2_TX},
 	{STM32_PIN_PA3, STM32F4_PINMUX_FUNC_PA3_USART2_RX},
 #endif	/* CONFIG_UART_STM32_PORT_2 */
+#ifdef CONFIG_SPI_0
+	{STM32_PIN_PA4, STM32F4_PINMUX_FUNC_PA4_SPI1_NSS},
+	{STM32_PIN_PA5, STM32F4_PINMUX_FUNC_PA5_SPI1_SCK},
+	{STM32_PIN_PA6, STM32F4_PINMUX_FUNC_PA6_SPI1_MISO},
+	{STM32_PIN_PA7, STM32F4_PINMUX_FUNC_PA7_SPI1_MOSI},
+#endif	/* CONFIG_SPI_0 */
+#ifdef CONFIG_SPI_1
+	{STM32_PIN_PB12, STM32F4_PINMUX_FUNC_PB12_SPI2_NSS},
+	{STM32_PIN_PB13, STM32F4_PINMUX_FUNC_PB13_SPI2_SCK},
+	{STM32_PIN_PB14, STM32F4_PINMUX_FUNC_PB14_SPI2_MISO},
+	{STM32_PIN_PB15, STM32F4_PINMUX_FUNC_PB15_SPI2_MOSI},
+#endif	/* CONFIG_SPI_1 */
 };
 
 static int pinmux_stm32_init(struct device *port)
